In [ ]:
print('hello')

hello


In [ ]:
print("Hello")

Hello


In [ ]:
print('Hello word')

Hello word


In [ ]:
print("Hello World")

Hello World


In [ ]:
print('Hello_python')

Hello_python


In [ ]:
print("Hello_python")

Hello_python


In [ ]:
print("Merry X'mas")

Merry X'mas


In [ ]:
print('Merry X\'mas')

Merry X'mas


In [ ]:
print('I want to ask you "Why don\'t you drive to work ?"')

I want to ask you "Why don't you drive to work ?"


In [ ]:
print("I want to ask you \"Why don't you drive to work ?\"")

I want to ask you "Why don't you drive to work ?"


In [ ]:
print('"I don\'t have a car\"')

"I don't have a car"


In [ ]:
print("\"I don 't have a car\"")

"I don 't have a car"


In [ ]:
print("You got a new job !? That's so exciting !")

You got a new job !? That's so exciting !


In [ ]:
print('You got a new job !? That\'s so exciting')

You got a new job !? That's so exciting


In [ ]:
print('สวัสดีวันจันทร์')

สวัสดีวันนจันทร์


In [ ]:
print("สวัสดีวันจันทร์")

สวัสดีวันจันทร์


In [ ]:
print('ความแตกต่างระหว่างคนเก่งกับคนไม่เก่ง คือ "การใช้เวลาว่างให้เป็นประโยชน์"')

ความแตกต่างระหว่างคนเก่งกับคนไม่เก่ง คือ "การใช้เวลาว่างให้เป็นประโยชน์"


In [ ]:
print("ความแตกต่างกับคนเก่งกับคนไม่เก่ง คือ การใช้เวลาว่างให้เป็นประโยชน์")

ความแตกต่างกับคนเก่งกับคนไม่เก่ง คือ การใช้เวลาว่างให้เป็นประโยชน์


In [ ]:
print('/\\/\\/\\')

/\/\/\


In [ ]:
print("/\\/\\/\\")

/\/\/\


In [ ]:
print('''a
an
ant''')

a
an
ant


In [ ]:
print("""a
an
ant""")

a
an
ant


In [ ]:
print('''\t*
*\t*\t*
\t*''')

	*
*	*	*
	*


In [ ]:
print("""\t*
*\t*\t*
\t*""")

	*
*	*	*
	*


In [ ]:
print('''*\t+\t*
+\t*\t+
*\t+\t*''')

*	+	*
+	*	+
*	+	*


In [ ]:
print("""*\t+\t*
+\t*\t+
*\t+\t*""")

*	+	*
+	*	+
*	+	*


In [ ]:
print("""Just because something
thinks differently from you,
does that mean it's not thinking ?""")

Just because something
thinks differently from you,
does that mean it's not thinking ?


In [ ]:
print('''Just becauhse somthing
thinks differently from you,
does that mean it's not thinking ?''')

Just becauhse somthing
thinks differently from you,
does that mean it's not thinking ?


In [ ]:
print('''\\\t\t/
\tx
/\t\t\\''')

\		/
	x
/		\


In [ ]:
print("""\\\t\t/
\tx
/\t\t\\""")

\		/
	x
/		\


# Print Numeric

In [ ]:
print(25)

25


In [ ]:
print('%d' %25)

25


In [ ]:
print('%f' %100)

100.000000


In [ ]:
print('%.6f' %100)

100.000000


In [ ]:
from math import pi
print('%.15f' %pi)

3.141592653589793


In [ ]:
import math
print('%.15f' %math.pi)

3.141592653589793


In [ ]:
a = 2
print(a)

2


In [ ]:
a = 12.5
print('%.1f' %a)

12.5


# Print String+Numeric

In [ ]:
a = 2
b = 3
print(a,'x',b,'=',a*b)

2 x 3 = 6


In [ ]:
a = 2
b = 3
print('%d x %d = %d' %(a,b, a*b))


2 x 3 = 6


In [ ]:
a = 2
b = 3
c = 5
print('%d*(%d + %d) = %d*%d + %d*%d' %(a,b,c,a,b,a,c))

2*(3 + 5) = 2*3 + 2*5


In [ ]:
a = 2
b = 3
c = 5
print("%d*(%d+%d) = %d*%d + %d*%d" %(a,b,c,a,b,a,c))

2*(3+5) = 2*3 + 2*5


In [ ]:
a = 2.4
b = 2.5
print('%.1f + %.1f = %.4f' %(a,b, a+b))

2.4 + 2.5 = 4.9000


In [ ]:
a = 2.4
b = 2.5
print("%.1f + %.1f = %.4f" %(a,b,a+b))

2.4 + 2.5 = 4.9000


In [ ]:
a = 5
b = 2
print('%.1f - %.2f = %.4f' %(a,b,a-b))

5.0 - 2.00 = 3.0000


In [ ]:
a = 5
b = 2
print('%.1f - %.2f = %.4f' %(a,b,a-b))

5.0 - 2.00 = 3.0000


In [ ]:
a = 5
b = 2
print("%.1f - %.2f = %.4f" %(a,b, a-b))

5.0 - 2.00 = 3.0000


In [ ]:
birthday = 25
print('ฉันเกิดวันที่', birthday, 'ธันวาคม')

ฉันเกิดวันที่ 25 ธันวาคม


In [ ]:
birthday = 25
print("ฉันเกิดวันที่",birthday, "ธันวาคม")

ฉันเกิดวันที่ 25 ธันวาคม


In [ ]:
a = 5
b = 100
print('%d เท่าของ %d มีค่าเท่ากับ %d' %(a, b, a*b))

5 เท่าของ 100 มีค่าเท่ากับ 500


In [ ]:
a = 3.5
print('เขามีเงินเยอะกว่าฉัน %.2f บาท' %a)

เขามีเงินเยอะกว่าฉัน 3.50 บาท


In [ ]:
a = 3.5
print("เขามีเงินเยอะกว่าฉัน %.2f บาท" %a)

เขามีเงินเยอะกว่าฉัน 3.50 บาท


In [ ]:
a = 5
print('ฉันมีกำไร',a,'%')

ฉันมีกำไร 5 %


In [ ]:
a = 5
print('ฉันได้กำไร %d %%' %a)

ฉันได้กำไร 5 %


In [ ]:
a = 2
b = 3.5
print('เมื่อวานฉันขาดทุน %d %% วันนี้ฉันได้กำไร %.2f %%' %(a,b))

เมื่อวานฉันขาดทุน 2 % วันนี้ฉันได้กำไร 3.50 %


In [ ]:
a = 2
b = 3.5
print("เมื่อวานฉันขาดทุน %d %% วันนี้ฉันได้กำไร %.2f %%" %(a,b))

เมื่อวานฉันขาดทุน 2 % วันนี้ฉันได้กำไร 3.50 %


เขียนโปรแกรมสร้างตัวแปรชื่อ numeric1 และกำหนดค่าให้เท่ากับ 5 หลังจากนั้นให้ทำารพิมพ์ค่าและชนิดข้อมูลของ numeric1 ออกมา

In [ ]:
numeric1 = 5
print (numeric1)
print(type(numeric1))

5
<class 'int'>


เขียนโปรแกรมสร้างตัวแปรชื่อ numeric2 และกำหนดค่าให้เท่ากับ -3 หลังจากนั้นให้ทำารพิมพ์ค่าและชนิดข้อมูลของ numeric2 ออกมา

In [ ]:
numeric2 = -3
print (numeric2)
print(type(numeric2))

-3
<class 'int'>


In [ ]:
numeric3 = 0
print(numeric3)
print(type(numeric3))

0
<class 'int'>


In [ ]:
numeric4 = int(input('Please insert integer:'))
print(numeric4)
print(type(numeric4))

Please insert integer:5
5
<class 'int'>


เขียนโปรแกรมสร้างตัวแปรชื่อ Numeric1 และกำหนดค่าให้เท่ากับ 5 หลังจากนั้นให้พิมพ์ค่า และชนิดของ Numeric1 ออกมา

In [ ]:
numeric1 = 5.0
print(numeric1)
print(type(numeric1))

5.0
<class 'float'>


เขียนโปรแกรมสร้างตัวแปรชื่อ Numeric2 และกำหนดค่าให้เท่ากับ -3.1 หลังจากนั้นให้พิมพ์ค่า และชนิดของ Numeric2 ออกมา

In [ ]:
numeric2 = -3.1
print(numeric2)
print(type(numeric2))

-3.1
<class 'float'>


เขียนโปรแกรมสร้างตัวแปรชื่อ Numeric3 และกำหนดค่าให้เท่ากับ 0.0 หลังจากนั้นให้พิมพ์ค่า และชนิดของ Numeric3 ออกมา

In [ ]:
numeric3 = 0.0
print(numeric3)
print(type(numeric3))

0.0
<class 'float'>


เขียนโปรแกรมรับอินพุต 1 ตัวที่เป้นจำนวจริง หลังจากนั้นให้ทำการพิมพ์ค่าและชนิดข้อมูลของจำนวนที่รับออกมา:



In [ ]:
numeric1 = int(input('please input:'))
print(numeric1)
print(type(numeric1))

please input:5
5
<class 'int'>


In [ ]:
เขียนโปรแกรมรับอินพุต 2 ตัวที่เป็นจำนวนต็ม(int1,int2) ให้คำนวณผลบวก,ผลลบ,ผลคูณ และผลหารของจำนวนที่รับมา หลังจากนั้นให้พิมพ์ค่าและชนิดข้อมูลของทุกผลลัพธ์ออกมา

In [ ]:
numeric1 = int(input('int1:'))
numeric2 = int(input('int2:'))
print('numeric1 =',numeric1,(type(numeric1)))
print('numeric2 =',numeric2,(type(numeric2)))
print('คำนวณผลบวก =',(numeric1+numeric2),(type(numeric1+numeric2)))
print('ผลลบ =',(numeric1-numeric2),(type(numeric1-numeric2)))
print('ผลคูณ =',(numeric1*numeric2),(type(numeric1*numeric2)))
print('ผลคูณ =',(numeric1/numeric2),(type(numeric1/numeric2)))

int1:100
int2:2
numeric1 = 100 <class 'int'>
numeric2 = 2 <class 'int'>
คำนวณผลบวก = 102 <class 'int'>
ผลลบ = 98 <class 'int'>
ผลคูณ = 200 <class 'int'>
ผลคูณ = 50.0 <class 'float'>


เขียนโปรแกรมรับอินพุต 2 ตัวที่เป็นจำนวนต็ม(float1,float2) ให้คำนวณผลบวก,ผลลบ,ผลคูณ และผลหารของจำนวนที่รับมา หลังจากนั้นให้พิมพ์ค่าและชนิดข้อมูลของทุกผลลัพธ์ออกมา

In [ ]:
numeric1 = float(input('float1:'))
numeric2 = float(input('float2:'))
print('numeric1 =',numeric1,(type(numeric1)))
print('numeric2 =',numeric2,(type(numeric2)))
print('คำนวณผลบวก =',(numeric1+numeric2),(type(numeric1+numeric2)))
print('ผลลบ =',(numeric1-numeric2),(type(numeric1-numeric2)))
print('ผลคูณ =',(numeric1*numeric2),(type(numeric1*numeric2)))
print('ผลคูณ =',(numeric1/numeric2),(type(numeric1/numeric2)))

float1:1
float2:1
numeric1 = 1.0 <class 'float'>
numeric2 = 1.0 <class 'float'>
คำนวณผลบวก = 2.0 <class 'float'>
ผลลบ = 0.0 <class 'float'>
ผลคูณ = 1.0 <class 'float'>
ผลคูณ = 1.0 <class 'float'>


เขียนโปรแกรมรับอินพุต 2 ตัวที่เป็นจำนวนต็ม(float1,int1) ให้คำนวณผลบวก,ผลลบ,ผลคูณ และผลหารของจำนวนที่รับมา หลังจากนั้นให้พิมพ์ค่าและชนิดข้อมูลของทุกผลลัพธ์ออกมา

In [ ]:
numeric1 = float(input('float1:'))
numeric2 = int(input('int1:'))
print('numeric1 =',numeric1,(type(numeric1)))
print('numeric2 =',numeric2,(type(numeric2)))
print('คำนวณผลบวก =',(numeric1+numeric2),(type(numeric1+numeric2)))
print('ผลลบ =',(numeric1-numeric2),(type(numeric1-numeric2)))
print('ผลคูณ =',(numeric1*numeric2),(type(numeric1*numeric2)))
print('ผลคูณ =',(numeric1/numeric2),(type(numeric1/numeric2)))

float1:100
int1:2
numeric1 = 100.0 <class 'float'>
numeric2 = 2 <class 'int'>
คำนวณผลบวก = 102.0 <class 'float'>
ผลลบ = 98.0 <class 'float'>
ผลคูณ = 200.0 <class 'float'>
ผลคูณ = 50.0 <class 'float'>


เขียนโปรแกรมสร้่างตัวแปรชื่อ Logic1 และกำหนดค่าให้เท่ากับ true หลังจากนั้นให้ทำการพิมพืค่าและชนิดข้อมูลของ Logic1 ออกมา

In [ ]:
logic1 = True
print(logic1)
print(type(logic1))

True
<class 'bool'>


เขียนโปรแกรมสร้างตัวแปรชื่อ logic2 และกำหนดค่าให้เท่ากับ false หลังจากนั้นให้พิมพ์ค่าและชนิดข้อมุลของ logic2 ออกมา

In [ ]:
logic2 = False
print(logic2)
print(type(logic2))

False
<class 'bool'>


In [ ]:
logic1 = True
logic2 = False
print(logic1 and logic2)

False


In [ ]:
logic1 = True
logic2 = False
print(logic1 or logic2)

True


In [ ]:
complex1 = 1+2j
print(complex1,type(complex1))

(1+2j) <class 'complex'>


In [ ]:
complex1 = 1+2j
print(complex1.real)

1.0


In [ ]:
complex1 = 1+2j
print(complex1.imag)

2.0


## `06-07-2022` 50 ข้อ

# String

19.เขียนโปรแกรมสร้างตัวแปรชื่อ String1 และกำหนดค่าให้เท่ากับ 'Python' หลังจากนั้นให้ทำการพิมพ์ค่าและชนิดข้อมูลของ string1 ออกมา

In [ ]:
string1 = 'Python'
print(string1,(type(string1)))

Python <class 'str'>


20.เขียนโปรแกรมสร้างตัวแปรชื่อ string1 และกำหนดค่าให้เท่ากับ 'Python' หลังจากนั้นให้ทำการพิมพ์อักขระที่ index เป็น 0 ออกมา

In [ ]:
string1 = 'Python'
print(string1[0])

P


21.เขียนโปรแกรมสร้างตัวแปรชื่อ string1 และกำหนดค่าให้เท่ากับ 'Python' หลังจากนั้นให้ทำการพิมพ์อักขระที่ index เป็น 1 ออกมา

In [ ]:
string1 = 'Python'
print(string1[1])

y


22.เขียนโปรแกรมสร้างตัวแปรชื่อ string1 และกำหนดค่าให้เท่ากับ 'Python' หลังจากนั้นให้ทำการพิมพ์อักขระที่ index เป็น -1 ออกมา

In [ ]:
string1 = 'Python'
print(string1[-1])

n


23.เขียนโปรแกรมสร้างตัวแปรชื่อ string1 และกำหนดค่าให้เท่ากับ 'Python' หลังจากนั้นให้ทำการพิมพ์อักขระที่ index เป็น -2 ออกมา

In [ ]:
string1 = 'Python'
print(string1[-2])

o


24.เขียนโปรแกรมสร้างตัวแปรชื่อ string1 และกำหนดค่าให้เท่ากับ 'Python' หลังจากนั้นให้ทำการพิมพ์อักขระ 't' ออกมา โดยอ้างอิงตำแหน่งจาก string1

In [ ]:
string1 = 'Python'
print(string1[2])

t


25.เขียนโปรแกรมสร้างตัวแปรชื่อ string1 และกำหนดค่าให้เท่ากับ 'Python' หลังจากนั้นให้ทำการพิมพ์อักขระ 'h' ออกมา โดยอ้างอิงตำแหน่งจาก string1

In [ ]:
string1 = 'Python'
print(string1[3])

h


26.เขียนโปรแกรมสร้างตัวแปรชื่อ string1 และกำหนดค่าให้เท่ากับ 'Python' หลังจากนั้นให้ทำการพิมพ์สายอักขระตั้งแต่ index ที่ 1 จนถึง index ที่ 4 ออกมา

In [ ]:
string1 = 'Python'
print(string1[1:4])

yth


27.เขียนโปรแกรมสร้างตัวแปรชื่อ string1 และกำหนดค่าให้เท่ากับ 'Python' หลังจากนั้นให้ทำการพิมพ์สายอักขระตั้งแต่ index ที่ 2 จนถึง index ที่ -1 ออกมา

In [ ]:
string1 = 'Python'
print(string1[2:-1])

tho


28.เขียนโปรแกรมสร้างตัวแปรชื่อ string1 และกำหนดค่าให้เท่ากับ 'Python' หลังจากนั้นให้ทำการพิมพ์สายอักขระตั้งแต่ index ที่ 2 จนถึง index ที่ -1 ออกมา

In [ ]:
string1 = 'Python'
print(string1[2:])

thon


29.เขียนโปรแกรมสร้างตัวแปรชื่อ string1 และกำหนดค่าให้เท่ากับ 'Python' หลังจากนั้นให้ทำการพิมพ์สายอักขระตั้งแต่ index ที่ -4 จนถึง index ที่ 5 ออกมา

In [ ]:
string1 = 'Python'
print(string1[-4:5])

tho


30.เขียนโปรแกรมสร้างตัวแปรชื่อ string1 และกำหนดค่าให้เท่ากับ 'Python' หลังจากนั้นให้ทำการพิมพ์สายอักขระตั้งแต่ index ที่ -6 จนถึง index ที่ 5 ออกมา

In [ ]:
string1 = 'Python'
print(string1[-6:5])

Pytho


In [ ]:
string1 = 'Python'
print(string1[:5])

Pytho


31.เขียนโปรแกรมสร้างตัวแปรชื่อ String1 และกำหนดค่าให้เท่ากับ 'Python' หลังจากนั้นให้ทำการพิมพ์สายอักขระตั้งแต่ index ที่ -4 จนถึง Index ที่ -2

In [ ]:
string1 = 'Python'
print(string1[-4:-1])

tho


32.เขียนโปรแกรมรับอินพุต 1 ตัวที่เป็นสายอักขระ จากนั้นให้ทำการพิมพ์ความยาวของสายอักขระ

In [ ]:
string = input('Please insert string:')
print(len(string1))

Please insert string:Okayyyy
6


33.เขียนโปรแกรมรับอินพุต 1 ตัวที่เป็นสายอักขระ ให้ทำการตรวจสอบสายอักขระที่รับว่า 'ก' หรือไม่ จากนั้นให้พิมพ์ค่าความจริงออกมา

In [ ]:
string = input('Please insert string:')
print('ก' in string)

Please insert string:กกหดก
True


34.เขียนโปรแกรมรับอินพุต 2 ตัวที่เป็นสายอักขระ (string1,string2)ให้ทำการตรวจ  String1 ที่รับมาว่ามี String2 ในนั้นหรือไม่ จากนั้นให้พิมพ์ค่าความจริงออกมา


> ตัวอย่างที่1:สมมติให้ string1 = 'happy',string2 = 'app' ผลลัพธ์ที่จะพิมพ์ออกมาคือ True



In [ ]:
string1 = input('Text1:')
string2 = input('Text2:')
print(string2 in string1
      )

Text1:12345
Text2:5
True


35.เขียนโปรแกรมรับอินพุต 1 ตัวที่เป็นสายอักขระ ให้ทำการแทนที่อักขระ 'a' ด้วย 'A' จากนั้นให้พิมพ์สายอักขระที่ผ่านดำเนินการแล้วออกมา

In [ ]:
string1 = input('Please insert string1:')
print(string1.replace('a','A'))

Please insert string1:a
A


36.เขียนโปรแกรมรับอินพุต 1 ตัวที่เป็นสายอักขระ ให้ทำการแทนที่อักขระ 'a' ด้วย 'A'และกำหนดที่ผ่านการดำเนินการใส่ตัวแปรชื่อ string2 จากนั้นให้พิมพ์ string2 ออกมา

In [ ]:
string1 = input('Please insert string1:')
string2 = string1.replace('a','A')
print(string2)

Please insert string1:aaa
AAA


37.เขียนโปรแกรมรับอินพุต 1 ตัวที่เป็นสายอักขระ (sentence) ให้ทำการตัดคำในสายอักขระโดยใช้ '' เป็นตัวตัดคำ หลังจากนั้นให้พิมพ์ผลลัพธ์จากการตัดคำออกมา


>ตัวอย่างที่ 1: สมมติให้ sentence = 'We love Python' ผลลัพธ์ที่จะพิมพ์ออกมาคือ['We','love','Python']



In [ ]:
sentence = input('Please insert sentence:')
print(sentence.split(' '))

Please insert sentence:We love Python
['We', 'love', 'Python']


38.เขียนโปรแกรมรับอินพุต 2 ตัวที่เป็นสายอักขระ (sentence, c) ให้ทำการตัดคำใน sentence โดยใช้ c เป็นตัวตัดคำ หลังจากนั้นให้พิมพ์ผลลัพธ์จากการการตัดคำออกมา


> ตัวอย่างที่1: สมมติให้ sentence = 'Py-thon',c = '-' ผลลัพธ์ที่จะพิมพ์ออกมาคือ ['Py','thon']



In [ ]:
string1 = input('Please insert string1:')
string2 = input('Please insert string2:')
print(string1.split(string2))

Please insert string1:sentence
Please insert string2:c
['senten', 'e']


39.เขียนโปรแกรมรับอินพุต 2 ตัวที่เป็นสายอักขระ (str1, str2) ให้ทำการต่อสายอักขระทั้งสอง (โดยนำ str2 ไปต่อหลัง str1) หลังจากนั้นให้พิมพ์ผลลัพธ์ออกมา


> ตัวอย่างที่ 1:สมมติให้ str1 = 'Fri',str2 ='day' ผลลัพธ์ที่จะพิมพ์ออกมาคือ 'friday'



In [ ]:
string1 = input('Please insert string1:')
string2 = input('Please insert string2:')
print(string1+string2)

Please insert string1:123-
Please insert string2:4567890
123-4567890


# List

40.เขียนโปรแกรมสร้าง List ที่มีสมาชิก 0,1,2,'a','b','c' ตามลำดับ จากนั้นให้ทำการพิมพ์ค่าและชนิดข้อมูลของ List ที่สร้างออกมา

In [ ]:
list1 = [0,1,2,'a','b','c']
print(list1)
print(type(list1))

[0, 1, 2, 'a', 'b', 'c']
<class 'list'>


41.กำหนดให้ list1 =[0,1,2,'a','b','c'] เขียนโปรแกรมเพื่อพิมพ์สมาชิกที่ inndex เป็น 2 ออกมา

In [ ]:
list1 =[0,1,2,'a','b','c']
print(list1[2])

2


42.กำหนดให้ list1 =[0,1,2,'a','b','c'] เขียนโปรแกรมเพื่อพิมพ์สมาชิกที่ inndex เป็น -3 ออกมา

In [ ]:
list1 =[0,1,2,'a','b','c']
print(list1[-3])

a


43.กำหนดให้ list1 =[0,1,2,'a','b','c'] เขียนโปรแกรมเพื่อพิมพ์ 0 ออกมาโดยอ้างอิงตำแหน่งจาก list1

In [ ]:
list1 =[0,1,2,'a','b','c'] 

In [ ]:
print(list1[0])

0


44.กำหนดให้ list1 =[0,1,2,'a','b','c'] เขียนโปรแกรมเพื่อพิมพ์ 'b'ออกมาโดยอ้างอิงตำแหน่งจาก list1

In [ ]:
list1 =[0,1,2,'a','b','c']
print(list1[4])

b


In [ ]:
list1 =[0,1,2,'a','b','c']
print(list1[-2])

b


45.กำหนดให้ list1 =[0,1,2,'a','b','c'] เขียนโปรแกรมเพื่อพิมพ์ list ที่มีสมาชิกตั้งแต่ index ที่ 2 จนถึง index ที่ 3 ของ list1 ออกมา

In [ ]:
list1 =[0,1,2,'a','b','c']
print(list1[2:4])

[2, 'a']


[1, 2, 'a', 'b']


46.กำหนดให้ list1 =[0,1,2,'a','b','c'] เขียนโปรแกรมเพื่อพิมพ์ list ที่มีสมาชิกตั้งแต่ index ที่ 1 จนถึง index ที่ -2 ของ list1 ออกมา

In [ ]:
list1 =[0,1,2,'a','b','c']
print(list1[1:-1])

[1, 2, 'a', 'b']


47.กำหนดให้ list1 =[0,1,2,'a','b','c'] เขียนโปรแกรมเพื่อพิมพ์[1,2,'a','b'] โดยอ้างอิงตำแหน่งจาก list1

In [ ]:
list1 =[0,1,2,'a','b','c']
print(list1[1:5])

[1, 2, 'a', 'b']


In [ ]:
list1 =[0,1,2,'a','b','c']
print(list1[1:-1])

[1, 2, 'a', 'b']


In [ ]:
list1 =[0,1,2,'a','b','c']
print(list1[-5:5])

[1, 2, 'a', 'b']


In [ ]:
list1 =[0,1,2,'a','b','c']
print(list1[-5:-1])

[1, 2, 'a', 'b']


48.กำหนดให้ list1 =[0,1,2,'a','b','c'] เขียนโปรแกรมเพื่อพิมพ์['a','b','c'] โดยอ้างอิงตำแหน่งจาก list1

In [ ]:
list1 =[0,1,2,'a','b','c']
print(list1[3:])

['a', 'b', 'c']


In [ ]:
list1 =[0,1,2,'a','b','c']
print(list1[-3:])

['a', 'b', 'c']


49.กำหนดให้ list1 =[0,1,2,'a','b','c'] เขียนโปรแกรมเพื่อพิมพ์[0,1,2] โดยอ้างอิงตำแหน่งจาก list1

In [ ]:
list1 =[0,1,2,'a','b','c']
print(list1[:3])

[0, 1, 2]


In [ ]:
list1 =[0,1,2,'a','b','c']
print(list1[:-3])

50.กำหนดให้ list2 =['ant','bird','cat','dog','eagle'] เขียนโปรแกรมเพื่อเปลี่ยนค่าสมาชิกใน index ที่ 3 เป็น 'duck' หลังจากนั้นให้พิมพ์ list2 ออกมา

In [ ]:
 list2 =['ant','bird','cat','dog','eagle']
 list2[3]= 'duck'
 print(list2)

['ant', 'bird', 'cat', 'duck', 'eagle']


51.กำหนดให้ list2 =['ant','bird','cat','dog','eagle'] เขียนโปรแกรมเพื่อเปลี่ยน  'ant'เป็น 'ape' หลังจากนั้นให้พิมพ์ list2 ออกมา

In [ ]:
 list2 =['ant','bird','cat','dog','eagle']
 list2[0]= 'ape'
 print(list2)

['ape', 'bird', 'cat', 'dog', 'eagle']


In [ ]:
 list2 =['ant','bird','cat','dog','eagle']
 list2[-5]= 'ape'
 print(list2)

['ape', 'bird', 'cat', 'dog', 'eagle']


52.กำหนดให้ list2 =['ant','bird','cat','dog','eagle']เขียนโปรแกรมเพื่อเพิ่ม 'fish' ไปเป้นสมาชิกตัวสุดท้ายของ list2 หลังจากนั้นให้่พิมพ์ list2 ออกมา

In [ ]:
 list2 =['ant','bird','cat','dog','eagle']
 list2[4]= 'fish'
 print(list2)

['ant', 'bird', 'cat', 'dog', 'fish']


In [ ]:
 list2 =['ant','bird','cat','dog','eagle']
 list2[-1]= 'fish'
 print(list2)

['ant', 'bird', 'cat', 'dog', 'fish']


53.เขียนโปรแกรมสร้าง empty list และรับอินพุฒ 3 ตัวที่เป็นสายอักขระ(str1,str2,str3) จากนั้นให้ทำการเพิ่มสายอักขระทั้ง 3 ไปเป็นสมาชิกใน list โดยใช้เมธอด append สุดท้ายให้พิมพ์ list ออกมา

In [ ]:
list2 = []
str1 = input('Please insert first element:')
str2 = input('Please insert second element:')
str3 = input('Please insert third element:')
list2.append(str1)
list2.append(str2)
list2.append(str3)
print(list2)

Please insert first element:1
Please insert second element:2
Please insert third element:3
['1', '2', '3']


54.กำหนดให้ list3 =['apple','cherry','eggfruit'] เขียนโปรแกรมเพื่อแทรก 'ba-nana' ไปเป็นสมชิกใน index ที่1 ของ list3 หลังจากนั้นให้พิมพ์ list3 ออกมา

In [ ]:
list3 =['apple','cherry','eggfruit']
list3.insert(1,'banana')
print(list3)

['apple', 'banana', 'cherry', 'eggfruit']


55.กำหนดให้ list3 =['apple','banana',''cherry','eggfruit'] เขียนโปรแกรมเพื่อแทรก 'kiwi' ไปเป็นสมชิกใน index ที่-2 ของ list3 หลังจากนั้นให้พิมพ์ list3 ออกมา


In [ ]:
list3 =['apple','cherry','eggfruit']
list3.insert(-1,'kiwi')
print(list3)

['apple', 'cherry', 'kiwi', 'eggfruit']


56.กำหนดให้ list4=[0,4,2,3,1] เขียนโปรแกรมเพื่อเรียงลำดับข้อมูลใน list4 จากน้อยไปมาก หลังจากนั้นให้พิมพ์ list4 ออกมา

In [ ]:
list4=[0,4,2,3,1] 
list4.sort()
print(list4)

[0, 1, 2, 3, 4]


57.กำหนดให้ list4=[0,4,2,3,1] เขียนโปรแกรมเพื่อเรียงลำดับข้อมูลใน list4 จากมากไปน้อย หลังจากนั้นให้พิมพ์ list4 ออกมา

In [ ]:
list4=[0,4,2,3,1] 
list4.sort(reverse=True)
print(list4)

[4, 3, 2, 1, 0]


58.กำหนดให้ list4=[0,4,2,3,1] เขียนโปรแกรมเพื่อเรียงลำดับข้อมูลใน list4 จากน้อยไปมาก และเก็บค่าไว้ในตัวแปรชื่อ sorted_list4 หลังจากนั้นให้พิมพ์ sorted_list4 ออกมา

In [ ]:
list4=[0,4,2,3,1]
sort_list4 = sorted(list4)
print(sort_list4)

[0, 1, 2, 3, 4]


59.กำหนดให้ list4=[0,4,2,3,1] เขียนโปรแกรมเพื่อเรียงลำดับข้อมูลใน list4 จากมากไปน้อย และเก็บค่าไว้ในตัวแปรชื่อ sorted_list4 หลังจากนั้นให้พิมพ์ sorted_list4 ออกมา

In [ ]:
list4=[0,4,2,3,1]
sort_list4 = sorted(list4,reverse=True)
print(sort_list4)

[4, 3, 2, 1, 0]


60.กำหนดให้ list5=['d','a','c','b','e'] เขียนโปรแกรมเพื่อเรียงลำดับข้อมูลใน list5 ตามตัวอักษร หล้ังจากนั้นให้พิมพ์ list5 ออกมา

In [ ]:
 list5=['d','a','c','b','e'] 
 list5.sort()
 print(list5)

['a', 'b', 'c', 'd', 'e']


61.กำหนดให้ list5=['d','a','c','b','e'] เขียนโปรแกรมเพื่อเรียงลำดับข้อมูลใน List5 ตามตัวอักษรแบบถอยหลัง และเก็บค่าไว้ในตัวแปรชื่อ sorted_list5 หลังจากนั้นให้พิมพ์ sorted_list5 ออกมา

In [ ]:
list5 = ['d','a','c','b','e']
sorted_list5 = sorted(list5, reverse = True)
print(sorted_list5
      )

['e', 'd', 'c', 'b', 'a']


62.กำหนดให้ list1 =['a','b','c'] และ list2 = [0,1,2] เขียนโปรแกรมเพื่อนำสมาชิกทั้งหมดของ list2 ไปต่อท้าย list1 จากนั้นให้พิมพ์ list1 และ list2 ออกมา

In [ ]:
list1 =['a','b','c']
list2 = [0,1,2]
list1.extend(list2)
print(list1,list2)

['a', 'b', 'c', 0, 1, 2] [0, 1, 2]


63.กำหนดให้ list1 =['a','b','c'] และ list2 = [0,1,2] เขียนโปรแกรมเพื่อนำสมาชิกทั้งหมดของ list2 ไปต่อท้าย list1 และเก็บ list ที่ได้จากการดำเนินการไว้ในตัวแปรชื่อ list3 จากนั้นให้พิมพ์ list1,list2 และ list3 ออกมา

In [ ]:
list1 =['a','b','c']
list2 = [0,1,2] 
list3 =list1+list2
print(list1)
print(list2)
print(list3)


['a', 'b', 'c']
[0, 1, 2]
['a', 'b', 'c', 0, 1, 2]


64.กำหนดให้ list1 =[1,2,3,1,2,3] เขียนโปรแกรมลบสมาชิกใน index ที่ 1 ออกจาก list1 โดยใช้คำสั่ง del หลังจากนั้นให้พิมพ์ list1 ออกมา

In [ ]:
list1 =[1,2,3,1,2,3]
del list1[1]
print(list1)

[1, 3, 1, 2, 3]


65.กำหนดให้ list1 = [1,2,3,1,2,3]  เขียนโปรแกรมลบ 3 ทั้งหมดออกจาก list1 โดยใช้คำสั่ง Del หลังจากนั้นให้พิมพ์ list1 ออกมา

In [ ]:
list1 =[1,2,3,1,2,3]
del list1[-1]
del list1[2]
print(list1)

[1, 2, 1, 2]


66.กำหนดให้ list1 = [1,2,3,1,2,3]  เขียนโปรแกรมลบ 1 ออกจาก list1 โดยใช้เมธอด Remove หลังจากนั้นให้พิมพ์ list1 ออกมา

In [ ]:
list1 = [1,2,3,1,2,3] 
list1.remove(1)
print(list1)

[2, 3, 1, 2, 3]


67.กำหนดให้ list1 = [1,2,3,1,2,3]  เขียนโปรแกรมลบ 1 ทั้งหมดออกจาก list1 โดยใช้เมธอด Remove หลังจากนั้นให้พิมพ์ list1 ออกมา


In [ ]:
list1 = [1,2,3,1,2,3] 
list1.remove(1)
list1.remove(1)
print(list1)

[2, 3, 2, 3]


68.กำหนดให้ list1 = [1,2,3,1,2,3]  เขียนโปรแกรมลบสมาชิกทุกตัวออกจาก list1 หลังจากนั้นให้พิมพ์ list1 ออกมา

In [ ]:
list1 = [1,2,3,1,2,3] 
list1.clear()
print(list1)

[]


## `07-07-2022` 50 ข้อ

69.กำหนดให้ list1 =[1,2,3,'a','b','c'] เขียนโปรแกรมหาความยาวของ list1 และพิมพ์ออกมา

In [ ]:
list1 =[1,2,3,'a','b','c']
print(len(list1))

6


70.กำหนดให้ list1 = [1,2,3,4,5] เขียนโปรแกรมรับอินพุต 1 ตัวที่เป็นจำนวนเต็ม(numeric1) ให้ทำการตรวจว่า numeric1 เป็นสมาชิกใน list1 หรือไม่ จากนั้นให้พิมพ์ค่าความจริงออกมา

In [ ]:
list1 = [1,2,3,4,5] 
numeric1 = int(input('Please insert numeric1:'))
print(numeric1 in list1)

Please insert numeric1:5
True


# **Tuple**

71. เขียนโปรแกรมสร้าง tuple ที่มีสมาชิกเป็น 'America','Brazil','China',Do-minican','Egypt' ตามลำกดับจากนั้นให้ทำการพิมพ์ค่า และชนิดข้อมูลของ tuple ที่สร้างออกมา

In [ ]:
tuple1=( 'America','Brazil','China','Do-minican','Egypt')
print(tuple1)
print(type(tuple1))


('America', 'Brazil', 'China', 'Do-minican', 'Egypt')
<class 'tuple'>


72.กำหนดให้ tuple1 =('America','Brazil','China','Do-minican','Egypt') เขียนโปรแกรมเพื่อพิมพ์สมาชิกที่ Index เป็น 1 ออกมา

In [ ]:
tuple1 =('America','Brazil','China','Do-minican','Egypt') 
print(tuple1[1])

Brazil


73.กำหนดให้ tuple1 =('America','Brazil','China','Do-minican','Egypt') เขียนโปรแกรมเพื่อพิมพ์สมาชิกที่ Index เป็น -2 ออกมา

In [ ]:
tuple1 =('America','Brazil','China','Do-minican','Egypt')
print(tuple1[-2])

Do-minican


74.กำหนดให้ tuple1 =('America','Brazil','China','Do-minican','Egypt')เขียนโปรแกรมเพื่อพิมพ์ 'China'ออกมาโดยอิ้างอิงตำแหน่งจาก tuple1

In [ ]:
tuple1 =('America','Brazil','China','Do-minican','Egypt')
print(tuple1[2])

China


In [ ]:
tuple1 =('America','Brazil','China','Do-minican','Egypt')
print(tuple1[-3])

China


75.กำหนดให้ tuple1 =('America','Brazil','China','Do-minican','Egypt')เขียนโปรแกรมเพื่อพิมพ์ 'Egypt'ออกมาโดยอิ้างอิงตำแหน่งจาก tuple1

In [ ]:
tuple1 =('America','Brazil','China','Do-minican','Egypt')
print(tuple1[-1])

Egypt


In [ ]:
tuple1 =('America','Brazil','China','Do-minican','Egypt')
print(tuple1[4])

Egypt


76.กำหนดให้ tuple2 =('one','two','three','1','2','3') เขียนโปรแกรมเพื่อพิมพ์ tuple ที่มีสมาชิกตั้งแต่ Index ที่ 1 จนถึง Index ที่3 ของ tuple2 ออกมา

In [ ]:
tuple2 =('one','two','three','1','2','3')
print(tuple2[1:4])

('two', 'three', '1')


77.กำหนดให้ tuple2 =('one','two','three','1','2','3')เขียนโปรแกรมเพื่อพิมพ์ tuple ที่มีสมาชิกตั้งแต่ Index ที่ 2 จนถึง Index ที่-2 ของ tuple2 ออกมา

In [ ]:
tuple2 =('one','two','three','1','2','3')
print(tuple2[2:-1])

('three', '1', '2')


78.กำหนดให้ tuple2 =('one','two','three','1','2','3')เขียนโปรแกรมเพื่อพิมพ์ ('three',1) โดยอ้างอิงตำแหน่งจาก tuple2

In [ ]:
tuple2 =('one','two','three','1','2','3')
print(tuple2[2:4])

('three', '1')


In [ ]:
tuple2 =('one','two','three','1','2','3')
print(tuple2[-4:-2])

('three', '1')


In [ ]:
tuple2 =('one','two','three','1','2','3')
print(tuple2[2:-2])

('three', '1')


In [ ]:
tuple2 =('one','two','three','1','2','3')
print(tuple2[-4:-2])

('three', '1')


79.กำหนดให้ tuple2 =('one','two','three','1','2','3')เขียนโปรแกรมเพื่อพิมพ์ ('one','two','three') โดยอ้างอิงตำแหน่งจาก tuple2

In [ ]:
tuple2 =('one','two','three','1','2','3')
print(tuple2[:3])

('one', 'two', 'three')


In [ ]:
tuple2 =('one','two','three','1','2','3')
print(tuple2[:-3])

('one', 'two', 'three')


In [ ]:
tuple2 =('one','two','three','1','2','3')
print(tuple2[0:3])

('one', 'two', 'three')


In [ ]:
tuple2 =('one','two','three','1','2','3')
print(tuple2[-6:-3])

('one', 'two', 'three')


80.กำหนดให้ tuple2 =('one','two','three','1','2','3')เขียนโปรแกรมเพื่อพิมพ์ (1,2,3) โดยอ้างอิงตำแหน่งจาก tuple2

In [ ]:
tuple2 =('one','two','three','1','2','3')
print(tuple2[3:])

('1', '2', '3')


In [ ]:
tuple2 =('one','two','three','1','2','3')
print(tuple2[-3:])

('1', '2', '3')


81.กำหนดให้ tuple3 =('one','two','three','four','five')เขียนโปรแกรมหาความยาวของ tuple3 และพิมพ์ออกมา

In [ ]:
tuple3 =('one','two','three','four','five')
print(len(tuple3))


5


82.กำหนดให้ tuple3 =('one','two','three','four','five')เขียนโปรแกรมรับอินพุต 1 ตัวที่เป็นสายอักขระ(str1) ให้ทำการตรวจว่า str1 เป็นสมาชิกใย tuple3 หรือไม่ จากนั้นให้พิมพ์ค่าความจริงออกมา

In [ ]:
tuple3 =('one','two','three','four','five')
str1 = str(input('Please insert str:'))
print(str1 in tuple3)

Please insert str:four
True


# **Dictionany**

83.เขียนโปรแกรมสร้าง dictionary ที่มี key เป็น 'first_name' และ 'last_name' โดยมี value เป็น 'John' และ 'Doe' ตามลำดับ จากนั้นให้ทำการพิมพ์ค่าและชนิดข้อมูลของ Dictionary ที่สร้างออกมา

In [ ]:
dict1 ={'first_name':'John','last_name' :'Doe'}
print(dict1)
print(type(dict1))

{'first_name': 'John', 'last_name': 'Doe'}
<class 'dict'>


84.กำหนดให้ dict1 = {'first_name':'John', 'last_name': 'doe'} เขียนโปรแกรมพิมพ์ value ของ key 'first_name' ออกมา

In [ ]:
dict1 = {'first_name':'John', 'last_name': 'doe'}
print(dict1['first_name'])

John


85.กำหนดให้ dict1 = {'first_name': 'John', 'last_name':'Doe'} เขียนโปรแกรมพิมพ์ value ของ key 'last_name' ออกมา

In [ ]:
dict1 = {'first_name':'John', 'last_name': 'doe'}
print(dict1['last_name'])

doe


86.กำหนดให้ dict1 = {'first_name':'John','last_name':'Doe'} เขียนโปรแกรมพิมพ์ list ของ key ทั้งหมดใน Dict1 ออกมา

In [ ]:
dict1 = {'first_name':'John', 'last_name': 'doe'}
print(dict1.keys())

dict_keys(['first_name', 'last_name'])


87.กำหนดให้ dict1 = {'first_name':'John','last_name':'Doe'} เขียนโปรแกรมพิมพ์ list ของ value ทั้งหมดใน Dict1 ออกมา

In [ ]:
dict1 = {'first_name':'John', 'last_name': 'doe'}
print(dict1.values())

dict_values(['John', 'doe'])


88.กำหนดให้ dict1 = {'first_name':'John','last_name':'Doe'} เขียนโปรแกรมเปลี่ยน value ของ key 'first_name' เป็น 'Jane' หลังจากนั้นให้พิมพ์ dict1 ออกมา

In [ ]:
dict1 = {'first_name':'John', 'last_name': 'doe'}
dict1['first_name']='Jane' 
print(dict1)

{'first_name': 'Jane', 'last_name': 'doe'}


89.กำหนดให้ dict1 = {'first_name':'John','last_name':'Doe'} เขียนโปรแกรมสร้าง key ใหม่ชื่อ 'age' และกำหนด value เป็น 32 หลังจากนั้นให้พิมพ์ dict1 ออกมา

In [ ]:
dict1 = {'first_name':'John', 'last_name': 'doe'}
dict1['age']=31
print(dict1)

{'first_name': 'John', 'last_name': 'doe', 'age': 31}


90.กำหนดให้ dict1 = {'first_name': 'John','last_name': 'Doe'} เขียนโปรแกรมสร้าง key ใหม่ชื่อ 'age' และ 'hobby' โดยมี value เป็น 32 และ ['coding','stucding'] ตามลำดับ หลังจากนั้นให้พิมพ์ dict1 ออกมา

In [ ]:
dict1 = {'first_name': 'John','last_name': 'Doe'} 
dict1.update({'age':32,'hobby':['coding','stucding']})
print(dict1)

{'first_name': 'John', 'last_name': 'Doe', 'age': 32, 'hobby': ['coding', 'stucding']}


91.เขียนโปรแกรมสร้าง empty dictionary และรับอินพุต 2 ตัวที่เป็นสายอักขระ (str1,str2) โดยให้ str1 เป็น key และ str2 เป็น value หลังจากนั้นให้พิมพ์ dictionary ออกมา

In [ ]:
dict1 = {}
str1 = input('Please insert str1:')
str2 = input('Please insert str2:')
dict1[str1]=str2
print(dict1)

Please insert str1:Name
Please insert str2:San
{'Name': 'San'}


92.กำหนดให้ dict1 ={'firstname':'John','last_name':'Doe','age':32} เขียนโปรแกรมลบ key 'age' ออกจาก dict1 หลังจากนั้นให้พมพ์ dict1 ออกมา

In [ ]:
dict1 ={'firstname':'John','last_name':'Doe','age':32}
del dict1['age']
print(dict1)

{'firstname': 'John', 'last_name': 'Doe'}


93.กำหนดให้ dict1 = {'first_name':'John','last_name':'Doe','age':32} เขียนโปรแกรมลบข้อมูลทั้งหมดออกจาก dict1 หลังจากนั้นให้พิมพ์ dict1 ออกมา

In [ ]:
dict1 ={'firstname':'John','last_name':'Doe','age':32}
dict1.clear()
print(dict1)

{}


94.กำหนดให้ dict1 = {'first_name':'John','last_name':'Doe','age':32} เขียนโปรแกรมพิมพ์จำนวน key ของ dict1 ออกมา

In [ ]:
dict1 = {'first_name':'John','last_name':'Doe','age':32}
print(len(dict1))

3


In [ ]:
dict1 = {'first_name':'John','last_name':'Doe','age':32}
print(len(dict1.keys()))

3


95.กำหนดให้ dict1 = {'first_name':'John','last_name':'Doe','age':32} เขียนโปรแกรมรับอินพุต 1 ตัวที่เป็นสายอักขระ (str1) ให้ทำการตรวจว่า str1 เป็นหนึ่งใน key ของ dict1 หรือไม่ จากนั้นให้พิมพ์ค่าความจริงออกมา

In [ ]:
dict1 = {'first_name':'John','last_name':'Doe','age':32} 
str1 = input('Please insert st1:')
print(str1 in dict1.keys())

Please insert st1:first_name
True


In [ ]:
dict1 = {'first_name':'John','last_name':'Doe','age':32} 
str1 = input('Please insert st1:')
print(str1 in dict1)

Please insert st1:first_name
True


96.กำหนดให้ dict1 = {'first_name':'John','last_name':'Doe','age':32} เขียนโปรแกรมรับอินพุต 1 ตัวที่เป็นสายอักขระ (str1) ให้ทำการตรวจว่า str1 เป็นหนึ่งใน value ของ dict1 หรือไม่ จากนั้นให้พิมพ์ค่าความจริงออกมา

In [ ]:
dict1 = {'first_name':'John','last_name':'Doe','age':32} 
str1 = input('Please insert st1:')
print(str1 in dict1.values())

Please insert st1:Doe
True


### **Set**

97.เขียนโปรแกรมสร้าง set ที่มีสมาชิก 1,2,3,'i','ii','iii' จากนั้านให้ทำการพิมพ์ค่าและชนิดข้อมูลของ Set ที่สร้างออกมา

In [ ]:
set1 = {1,2,3,'i','ii','iii'}
print(set1)
print(type(set1))

{1, 2, 3, 'iii', 'ii', 'i'}
<class 'set'>


98.กำหนดให้ set1 ={1,2,3,'i','ii','iii'} เขียนโปรแกรมพิมพ์สมาชิกทุกตัวใน set1 ออกมา

In [ ]:
set1 = {1,2,3,'i','ii','iii'}
for s in set1 :
  print(s)

1
2
3
iii
ii
i


99.กำหนดให้ set1 ={1,2,3,'i','ii','iii'} เขียนโปรแกรมเพิ่ม 'iv' ไปเป็นสมาชิกใน set1 จากนั้นให้พิมพ์ set1 ออกมา

In [ ]:
set1 = {1,2,3,'i','ii','iii'}
set1.add('iv')
print(set1)

{1, 2, 3, 'iii', 'iv', 'ii', 'i'}


100.กำหนดให้ set1 ={1,2,3,'i','ii','iii'} เขียนโปรแกรมเพิ่ม 1 ไปเป็นสมาชิกใน set1 จากนั้นให้พิมพ์ set1 ออกมา

In [ ]:
set1 = {1,2,3,'i','ii','iii'}
set1.add(1)
print(set1)

{1, 2, 3, 'iii', 'ii', 'i'}


101.เขียนโปรแกรมสร้าง empty set และรับอินพุต 3 ตัวที่เป็นสายอักขระ(str1,str2,str3) จากนั้นให้ทำการเพิ่มสายอักขระทั้ง 3 ไปเป็นสมาชิกใน set สุดท้ายให้พิมพ์ set ออกมา

In [ ]:
set1 = set()
str1 = input('Please insert str1:')
str2 = input('Please insert str2:')
str3 = input('Please insert str3:')
set1.add(str1)
set1.add(str2)
set1.add(str3)
print(set1)

Please insert str1:1
Please insert str2:2
Please insert str3:3
{'2', '3', '1'}


102.เขียนโปรแกรมสร้าง empty set และรับอินพุต 3 ตัวที่เป็นสายอักขระ(str1,str2,str3) จากนั้นให้ทำการเพิ่มสายอักขระทั้ง 3 ไปเป็นสมาชิกใน set โดยการใช้เมธอด update สุดท้ายให้พิมพ์ set ออกมา

In [ ]:
set1 = set()
str1 = input('Please insert str1:')
str2 = input('Please insert str2:')
str3 = input('Please insert str3:')
set1.update({str1,str2,str3})
print(set1)

Please insert str1:1
Please insert str2:2
Please insert str3:3
{'2', '3', '1'}


103.กำหนดให้ set1 ={1,2,3,'i','ii','iii'} เขียนโปรแกรมลบ 'i' จาก set1 จากนั้นให้พิมพ์ set1 ออกมา

In [ ]:
set1 ={1,2,3,'i','ii','iii'}
set1.remove('i')
print(set1)

{1, 2, 3, 'iii', 'ii'}


104.กำหนดให้ set1 ={1,2,3,'i','ii','iii'} เขียนโปรแกรมพิมพ์จำนวนสมาชิกใน set1 ออกมา

In [ ]:
set1 ={1,2,3,'i','ii','iii'}
print(len(set1))

6


105.กำหนดให้ set1 ={1,2,3,'i','ii','iii'} เขียนโปรแกรมรับอินพุต 1 ตัวที่เป็นจำนวนเต็ม(int1) ให้ทำการตรวจว่า int1 เป็นสมาชิกของ set1 หรือไม่ จากน้ั้นให้พิมพ์ค่าความจริงออกมา

In [ ]:
set1 ={1,2,3,'i','ii','iii'} 
int1 = int(input('Please insert int1:'))
print(int1 in set1)

Please insert int1:5
False


106.กำหนดให้ set1 ={1,2,3,4,5} และ set2 ={3,4,5,6} เขียนโปรแกรมพิมพ์ผลลัพธ์ที่เกิดจากการ union ระหว่าง set1 และ set2 ออกมา 

In [ ]:
 set1 ={1,2,3,4,5}
 set2 ={3,4,5,6}
 print(set1 | set2)


{1, 2, 3, 4, 5, 6}


107.กำหนดให้ set1 ={1,2,3,4,5} และ set2 ={3,4,5,6} เขียนโปรแกรมพิมพ์ผลลัพธ์ที่เกิดจากการ intersection ระหว่าง set1 และ set2 ออกมา

In [ ]:
 set1 ={1,2,3,4,5}
 set2 ={3,4,5,6}
 print(set1 & set2)

{3, 4, 5}


108.กำหนดให้ set1 ={1,2,3,4,5} และ set2 ={3,4,5,6} เขียนโปรแกรมพิมพ์ผลลัพธ์ที่เกิดจาก set1 - set2 ออกมา

In [ ]:
set1 ={1,2,3,4,5}
set2 ={3,4,5,6}
print(set1-set2)

{1, 2}


109.กำหนดให้ set1 ={1,2,3,4,5} และ set2 ={3,4,5,6} เขียนโปรแกรมพิมพ์ผลลัพธ์ที่เกิดจาก set2 - set1 ออกมา

In [ ]:
set1 ={1,2,3,4,5}
set2 ={3,4,5,6}
print(set2-set1)

{6}


110.กำหนดให้ set1 ={1,2,3,4,5} และ set2 ={3,4,5,6} เขียนโปรแกรมพิมพ์ผลลัพธ์ที่เกิดจากการ symmetric difference ระหว่าง set1 และ set2 ออกมา



In [ ]:
set1 ={1,2,3,4,5}
set2 ={3,4,5,6} 
print(set1 ^ set2)

{1, 2, 6}


1

6

111.

112.

113.

114.

115.

116.

117.

118.

# 08-07-2022 50 ข้อ

22-07-2022

In [ ]:
count_prime = 0
number = 1
while count_prime != 100:
    count = 0
    for i in range(1,number+1):
        if number%i == 0:
            count = count+1
        if count ==2:
            print(number)
            count_prime = count_prime +1
        number = number+1
    print(i)

SyntaxError: ignored

In [ ]:
for i in range(1,100):
    if i%3 == 0:
      print(i)

3
6
9
12
15
18
21
24
27
30
33
36
39
42
45
48
51
54
57
60
63
66
69
72
75
78
81
84
87
90
93
96
99


In [ ]:
for i in range(1,100):
  if (i%3 == 0) and (i%5 == 0):
    print(i)

15
30
45
60
75
90


In [ ]:
i = 1
while i <= 99:
  if (i%3 == 0) and (i%5 == 0):
    print(i)
  i = i+1

15
30
45
60
75
90


In [ ]:
for i in range(1,100):
  if (i%3 == 0) or (i%5 == 0):
    print(i)

3
5
6
9
10
12
15
18
20
21
24
25
27
30
33
35
36
39
40
42
45
48
50
51
54
55
57
60
63
65
66
69
70
72
75
78
80
81
84
85
87
90
93
95
96
99


In [ ]:
for i in range(1,1000):
  if (i%3 == 0) and (i%5 == 0) and (i%7 == 0):
    print(i)

105
210
315
420
525
630
735
840
945


In [ ]:
i = 1
while i <= 1000:
  if (i%3 == 0) and (i%5 == 0) and (i%7 == 0):
    print(i)
  i= i+1


105
210
315
420
525
630
735
840
945


In [ ]:
for i in range(1,99):
  if (i%3 ==0) or (i%5 == 0) or (i%7 == 0):
    print(i)

3
5
6
7
9
10
12
14
15
18
20
21
24
25
27
28
30
33
35
36
39
40
42
45
48
49
50
51
54
55
56
57
60
63
65
66
69
70
72
75
77
78
80
81
84
85
87
90
91
93
95
96
98


In [ ]:
for i in range(1,100):
  if i%3 ==0:
    if i%5 == 0:
      print(i)
  else:
    if i%5 == 0:
      print(i)

5
10
15
20
25
30
35
40
45
50
55
60
65
70
75
80
85
90
95


In [ ]:
for i in range(1,100):
  if (i%3 == 0) and (i%5 ==0):
    if i%2 != 0:
      print(i)


15
45
75


In [ ]:
for i in range(1,100):
  if (i%3 ==0) or (i%5 ==0):
    if i%2 !=0:
      print(i)

3
5
9
15
21
25
27
33
35
39
45
51
55
57
63
65
69
75
81
85
87
93
95
99


In [ ]:
count = 0
for i in range(1,1000):
  if i%3 ==0:
    count = count+1
    print(i)

3
6
9
12
15
18
21
24
27
30
33
36
39
42
45
48
51
54
57
60
63
66
69
72
75
78
81
84
87
90
93
96
99
102
105
108
111
114
117
120
123
126
129
132
135
138
141
144
147
150
153
156
159
162
165
168
171
174
177
180
183
186
189
192
195
198
201
204
207
210
213
216
219
222
225
228
231
234
237
240
243
246
249
252
255
258
261
264
267
270
273
276
279
282
285
288
291
294
297
300
303
306
309
312
315
318
321
324
327
330
333
336
339
342
345
348
351
354
357
360
363
366
369
372
375
378
381
384
387
390
393
396
399
402
405
408
411
414
417
420
423
426
429
432
435
438
441
444
447
450
453
456
459
462
465
468
471
474
477
480
483
486
489
492
495
498
501
504
507
510
513
516
519
522
525
528
531
534
537
540
543
546
549
552
555
558
561
564
567
570
573
576
579
582
585
588
591
594
597
600
603
606
609
612
615
618
621
624
627
630
633
636
639
642
645
648
651
654
657
660
663
666
669
672
675
678
681
684
687
690
693
696
699
702
705
708
711
714
717
720
723
726
729
732
735
738
741
744
747
750
753
756
759
762
765
768
771
774
777


In [ ]:
sumx = 0
for i in range(5):
  num =int(input('get input:'+str(i+1)))
  sumx = sumx+num
print(sumx)

get input:11
get input:22
get input:33
get input:44
get input:55
15


In [ ]:
sumx = 0
for i in range(1,1000):
  if i%3 ==0:
    sumx = sumx+1
  print(i)

1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
32
33
34
35
36
37
38
39
40
41
42
43
44
45
46
47
48
49
50
51
52
53
54
55
56
57
58
59
60
61
62
63
64
65
66
67
68
69
70
71
72
73
74
75
76
77
78
79
80
81
82
83
84
85
86
87
88
89
90
91
92
93
94
95
96
97
98
99
100
101
102
103
104
105
106
107
108
109
110
111
112
113
114
115
116
117
118
119
120
121
122
123
124
125
126
127
128
129
130
131
132
133
134
135
136
137
138
139
140
141
142
143
144
145
146
147
148
149
150
151
152
153
154
155
156
157
158
159
160
161
162
163
164
165
166
167
168
169
170
171
172
173
174
175
176
177
178
179
180
181
182
183
184
185
186
187
188
189
190
191
192
193
194
195
196
197
198
199
200
201
202
203
204
205
206
207
208
209
210
211
212
213
214
215
216
217
218
219
220
221
222
223
224
225
226
227
228
229
230
231
232
233
234
235
236
237
238
239
240
241
242
243
244
245
246
247
248
249
250
251
252
253
254
255
256
257
258
259
260
261
262
263
264
265
266
267
268
269
270
271
272
273
274
275
276
277


In [ ]:
sumx =0
for i in range(5):
  if i <=5:
    num =float(input('get input No:'+str(i+1)+' คือ '))
    sumx = num+1
    print(sumx)

get input No:1 คือ 1
2.0
get input No:2 คือ 2
3.0
get input No:3 คือ 3
4.0
get input No:4 คือ 4
5.0
get input No:5 คือ 5
6.0


In [ ]:
number = int(input('get input:'))
for i in  range(1,number+1):
  if number%i ==0:
    print(i)



get input:4
1
2
4


In [ ]:
for number in range(1,100):
  count = 0
  for i in range(1,number+1):
    if number%1 == 0:
      count = count+1
      if count ==2:
        print(number)


2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
32
33
34
35
36
37
38
39
40
41
42
43
44
45
46
47
48
49
50
51
52
53
54
55
56
57
58
59
60
61
62
63
64
65
66
67
68
69
70
71
72
73
74
75
76
77
78
79
80
81
82
83
84
85
86
87
88
89
90
91
92
93
94
95
96
97
98
99


In [ ]:
count_prime = 0
number = 1
while count_prime != 100:
  count = 0
  for i in range(1,number+1):
    if number%i ==0:
      count = count+1
  if count == 2:
      count_prime = count_prime+1
      print(number,'= จำนวนเฉพาะตัวที่',count_prime)
  number = number+1

2 = จำนวนเฉพาะตัวที่ 1
3 = จำนวนเฉพาะตัวที่ 2
5 = จำนวนเฉพาะตัวที่ 3
7 = จำนวนเฉพาะตัวที่ 4
11 = จำนวนเฉพาะตัวที่ 5
13 = จำนวนเฉพาะตัวที่ 6
17 = จำนวนเฉพาะตัวที่ 7
19 = จำนวนเฉพาะตัวที่ 8
23 = จำนวนเฉพาะตัวที่ 9
29 = จำนวนเฉพาะตัวที่ 10
31 = จำนวนเฉพาะตัวที่ 11
37 = จำนวนเฉพาะตัวที่ 12
41 = จำนวนเฉพาะตัวที่ 13
43 = จำนวนเฉพาะตัวที่ 14
47 = จำนวนเฉพาะตัวที่ 15
53 = จำนวนเฉพาะตัวที่ 16
59 = จำนวนเฉพาะตัวที่ 17
61 = จำนวนเฉพาะตัวที่ 18
67 = จำนวนเฉพาะตัวที่ 19
71 = จำนวนเฉพาะตัวที่ 20
73 = จำนวนเฉพาะตัวที่ 21
79 = จำนวนเฉพาะตัวที่ 22
83 = จำนวนเฉพาะตัวที่ 23
89 = จำนวนเฉพาะตัวที่ 24
97 = จำนวนเฉพาะตัวที่ 25
101 = จำนวนเฉพาะตัวที่ 26
103 = จำนวนเฉพาะตัวที่ 27
107 = จำนวนเฉพาะตัวที่ 28
109 = จำนวนเฉพาะตัวที่ 29
113 = จำนวนเฉพาะตัวที่ 30
127 = จำนวนเฉพาะตัวที่ 31
131 = จำนวนเฉพาะตัวที่ 32
137 = จำนวนเฉพาะตัวที่ 33
139 = จำนวนเฉพาะตัวที่ 34
149 = จำนวนเฉพาะตัวที่ 35
151 = จำนวนเฉพาะตัวที่ 36
157 = จำนวนเฉพาะตัวที่ 37
163 = จำนวนเฉพาะตัวที่ 38
167 = จำนวนเฉพาะตัวที่ 39
173 = จำนวนเฉพาะตัวที่ 4

**02082022**

In [ ]:
def addition(a,b):
  return a+b

In [ ]:
addition(18,32)

50

In [ ]:
def addition3(a,b,c):
  return a + b + c


In [ ]:
addition3(10,20,25)

55

In [ ]:
def square(a) :
  return a**2

In [ ]:
square(25)

625

In [ ]:
def square_list(listA):
  square = []
  for a in listA:
    square.append(a**2)
    return square

In [ ]:
square_list[1,2,3,4,5]

TypeError: ignored